spec/spec.md

```diff
@@ -307,10 +308,10 @@ Design Principle #5 ([Cryptographic Verifiability](https://trustoverip.org/perma
 
 VIDs can be divided into two subclasses as shown in figure 9:
 
```

Comment:
Suggestion: Define "external system"

Reply:
I'm deferring all VID subclass related issues to a later PR as needed.